### PR TITLE
[ADD][11.0] sale_stock_return_request: New module

### PR DESCRIPTION
**Pull request by chienandalu**
Depends on:
- [x] stock_return_request https://github.com/OCA/stock-logistics-workflow/pull/586

This module extends stock_return_request to be able to link the returns to
their corresponding purchase orders in the case of returns to supplier as well
as to be able to get the corresponding refunds.

cc @Tecnativa TT20569

### CODE REVIEW

File: sale_stock_return_request/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Stock Return Request",
+    "version": "11.0.1.0.0",
+    "category": "Stock",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
```
Comment (sergio-teruel) [inline sale_stock_return_request/__manifest__.py:14]: Add development_status key

File: sale_stock_return_request/models/stock_return_request.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+from odoo import fields, models
+
+
+class StockReturnRequest(models.Model):
+    _inherit = 'stock.return.request'
+
+    sale_order_ids = fields.Many2many(
+        comodel_name='sale.order',
+        string='Involved Sales',
+        readonly=True,
```
Comment (sergio-teruel) [inline sale_stock_return_request/models/stock_return_request.py:12]: In v12.0 is readonly by default.

**Review by sergio-teruel — COMMENTED** [review]
Please.. check action in tests to increase the coverage...

File: sale_stock_return_request/models/stock_return_request.py (hunk 2 above)
Comment (chienandalu) [reply sale_stock_return_request/models/stock_return_request.py:12]: But this is v11 :thinking:

Comment (sergio-teruel) [reply sale_stock_return_request/models/stock_return_request.py:12]: :astonished: ... Oks... Sorry

File: sale_stock_return_request/__manifest__.py (hunk 1 above)
Comment (chienandalu) [reply sale_stock_return_request/__manifest__.py:14]: done

**Comment by chienandalu** [comment]
> Please.. check action in tests to increase the coverage...

Done

**Review by sergio-teruel — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Labelling as WIP until the dependency is merged.

**Comment by chienandalu** [comment]
This can be merged already

**Comment by chienandalu** [comment]
Now it's green.

**Comment by chienandalu** [comment]
RTM

**Pull request merged**